src/components/PreMergeBanner.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import styled from "styled-components"
+import BannerNotification from "./BannerNotification"
+import Link from "./Link"
+
+const StyledBannerNotification = styled(BannerNotification)`
+  display: block;
+  z-index: 1;
+  a {
+    text-decoration: underline;
+  }
+`
+
+export interface IProps {
+  className?: string
+}
+
+const PreMergeBanner: React.FC<IProps> = ({ className }) => (
+  <StyledBannerNotification shouldShow className={className}>
+    The Merge is approaching which comes with many changes to Ethereum. Some
```

Comment:
Missing comma:
```suggestion
    The Merge is approaching, which comes with many changes to Ethereum. Some
```

Slight reword, if you prefer:
```suggestion
    The Merge is approaching, and with it comes many changes to Ethereum. Some
```